tests/phpunit-compare.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've always thought it was odd that the useragent tests were marked by the "COMPARE" flag. Is this because these tests evolved from the `compare` tests that `browscap-php` runs (where it compares `browscap-php` and `get_browser`)?

Should this flag/phpunit.xml file be named something else for this suite?

Reply:
The comparasation is made between the results written in out 15000 testcases and the real result returned the browscap-php.
This separate test case is nessesary to move out the test cases from the BrowscapTest directory to make code style checks faster for the other tests.

If you have a better name for this test step, please tell me.

Reply:
It's not that big of a deal really.

But a couple of suggestions:

`phpunit-integration.xml.dist`
`phpunit-useragents.xml.dist`

I think I prefer the first one, since they're really acting as integration tests.